src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -439,4 +504,109 @@ public boolean isSelected() {
             return selected;
         }
     }
+
+    public ProgressiveRendering getGeneratorByTaskId(@QueryParameter String taskId) throws Exception {
```

Comment:
```suggestion
    public ProgressiveRendering getGeneratorByTaskId(String taskId) throws Exception {
```
This is not a web method, or at least it is not being used as one.

Reply:
removed